crypto/build.gradle.kts

```diff
@@ -37,7 +37,7 @@ dependencies {
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:$jupiterVersion")
 }
 
-java{
+java {
```

Comment:
this change does not applied in tx/build.gradle.kts.

Reply:
I moved maven-related definition into publishing blocks in both tx/ and crypto/. 
Do you mean this??

Reply:
I mean line 18 in tx/buid.gradle.kts, there is no blank between `java{`.

Reply:
Ok~